rules/config

```diff
@@ -36,7 +36,7 @@ SONIC_CONFIG_MAKE_JOBS = $(shell nproc)
 DEFAULT_USERNAME = admin
 
 # DEFAULT_PASSWORD - default password for installer build
-DEFAULT_PASSWORD = YourPaSsWoRd
+DEFAULT_PASSWORD = admin123
```

Comment:
Let's have sonic password as default. It's sonic at the end...

Reply:
ok will revert